mixins/loading-complete/README.md

```diff
@@ -1,22 +1,47 @@
 # LoadingCompleteMixin
 
-The `LoadingCompleteMixin` contains boilerplate code for [`getLoadingComplete`](https://github.com/BrightspaceUI/testing#waiting-for-asynchronous-components). It simplifies fixture setup for custom components in automated tests.
+The `LoadingCompleteMixin` contains boilerplate code for [`getLoadingComplete`](https://github.com/BrightspaceUI/testing#waiting-for-asynchronous-components). It simplifies fixture setup for custom components in automated tests, and provides a `loadingComplete` promise that can be used internally.
 
 ## Usage
-Apply the mixin and call `resolveLoadingComplete()` once your component has finished loading to indicate it is ready for validation.
+Apply the mixin and call `resolveLoadingComplete()` once your component has finished loading
 
 ```js
 import { LoadingCompleteMixin } from '@brightspace-ui/core/mixins/loading-complete/loading-complete-mixin.js';
 
 class MyComponent extends LoadingCompleteMixin(LitElement) {
 
 	connectedCallback() {
-		this._fetchMyData().then(() => {
-			this.resolveLoadingComplete();
-		});
+		this._fetchMyData().then(this.resolveLoadingComplete);
 	}
 
 }
 ```
 
-If for whatever reason `resolveLoadingComplete` is never called, the `getLoadingComplete` promise will never resolve. In such cases any consumers (e.g. fixtures) will hang. If this behavior is not desired, ensure all code paths eventually call `resolveLoadingComplete`, or specify `{ awaitLoadingComplete: false }` in the `fixture` call.
+To make use of the `loadingComplete` promise, simply `await` it where needed
+
+```js
+async removeSkeleton() {
+	await this.loadingComplete;
+	this.skeleton = false;
+}
+```
+
+If for any reason `resolveLoadingComplete` is never called, `loadingComplete` won't resolve and any consumers will hang
+
+### `getLoadingComplete`
+
+In some cases, instead of finding one spot to call `resolveLoadingComplete`, you may find it easier to `await` a set of promises in a custom `getLoadingComplete` method
+
+You'll also need to use this method if you're working in a general-use mixin rather than directly in a component
+
+```js
+class MyComponent extends LoadingCompleteMixin(LitElement) {
+
+	async getLoadingComplete() {
+		await super.getLoadingComplete();
```

Comment:
Hopefully people who use this would recognize that this would be sequential, and if they want them all in parallel then they would do this instead:
```javascript
getLoadingComplete() {
    return Promise.all([
        super.getLoadingComplete(),
        this._myCustomIconImport,
        ...this._allMyDataPromises
    ]);
}
```

Reply:
These are just the promises, though, not the things that start the work and return the promises. However, maybe that's unclear here and we should just document it this way to avoid confusion.

Reply:
I see yeah. I wonder if anyone will ever be tempted to call a method here that actually kicks off work...?

Reply:
Yeah, we should definitely call that out better.